foundations/html_css/block-and-inline.md

```diff
@@ -28,7 +32,7 @@ Div is a block-level element by default. It is commonly used as a container elem
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-Span is an inline-level element by default. It is commonly used to group text content and inline HTML elements so we can apply styling to them.
+Span is an inline-level element by default. It is generally used to group text content and inline HTML elements to apply styling to them.
```

Comment:
I'm kinda feeling like "commonly" makes more sense here, partly because that's the verbiage used for the div line further up, and because "generally" seems *too* restrictive is the best I can describe the feeling 😆 

I'm gonna leave this open for another day to mull it over. If anyone else has thoughts feel free to comment, or if you have any persuasive argument for the change definitely feel free to comment as well, @lloydlobo

Reply:
The word _"commonly"_ does sound and feels to flow better with the overall context. Especially as you mentioned, it is being used along with `<div>` in the prior sentences.

After doing a quick research about the `<span>` element, I want to propose a new alternative. This would mean that neither _"commonly"_ nor _"generally"_ will be necessary in this case.

Newer Proposed Change:
>Span is an inline-level element by default. It groups text content and inline HTML elements for styling and is best used when no other HTML semantic element is available.

Since it's used as a last resort basis, it seems that its use could be kept to a bare minimum. As newer semantic elements are being introduced frequently, we may one day be able to do away with the good ol' `<span>`. :)

One downside that seems to be with the newer alternative is if the reader isn't familiar with semantic elements. What do you feel about that? 

We could go back with _"commonly"_ anyway. I feel this newer change may contribute to more complexity.

Reply:
Ah no I was just trying to separate out the two options 😆  I don't think we should link to anything in that verbiage.

Reply:
Okay that makes sense and clears my doubt :). I feel we should go with the `[Option B]`. 
It's more appropriate between the two options (pun intended). 

For now this would look like:
>Span is an inline-level element by default. It can be used to group text content and inline HTML elements for styling, and should only be used when no other HTML semantic element is appropriate.

<sub>Note: Added _"semantic"_ as an Easter Egg for the user.</sub>

Do you have any suggestions to add to this?

Reply:
I think the only thing I can think of would be a minor nitpick in reversing "HTML semantic" so that it reads "semantic HTML elements...", but this looks good! 💪🏼 Once you get a new commit pushed just re-request a review from me and I'll take another look over things.

Reply:
Done!